Formula/vim@7.4.rb

```diff
@@ -77,9 +77,18 @@ def install
       opts << "--without-x"
     end
 
-    if build.with? "luajit"
```

Comment:
I don't think we should be maintaining versioned formulae like this, nor is the change related to the PR.